records/scenario/scenario0001.json

```diff
@@ -0,0 +1,31 @@
+[
+    {
+        "@type": "Scenario",
+        "configuration": null,
+        "title": "FortiNDR Detections",
+        "description": "FortiNDR Cloud detects network anomalies and malicious content on your network.\n\nObserve the following\n* Detection Information (Status, Category, Confidence, Next steps etc) is presented as a handy reference\n* On receiving information from FortiNDR Cloud related to Detection FortiSOAR will create the Alert and after triggering Playbook \"Fetch Detection Events\", it will get all events related to that specific Detection and updates the indicators like IP Addresses, Files hashes etc.",
+        "steps": {
+            "Steps": [
+                {
+                    "name": "High Risk Typo Domain Response",
+                    "type": "executePlaybook",
+                    "playbookIRI": "\/api\/3\/workflows\/83dfbd95-2239-4a5f-96bc-b281d473e434",
+                    "preCreateMessage": "Creating demo alert record ",
+                    "postCreateMessage": "",
+                    "waitTimeforNextStep": "0"
+                }
+            ]
+        },
+        "createdAlertsID": "[\"\/api\/3\/alerts\/c8346414-6a30-43f6-bc48-c8675c7b0eb1\"]",
```

Comment:
createdAlertsID value should be null. Please remove alert IRI from the scenario record